src/index.html

```diff
@@ -1,13 +1,329 @@
 <!DOCTYPE html>
-<html lang="en">
+
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
+
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+
+    <title>MyBIKE</title>
+
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="../src/icons/favicon.png"
+    >
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
+
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page_body">
+    <header class="header">
+      <div class="header__content">
+        <div class="top-bar">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              src="images/logo.png"
+              alt="my bike logo"
+              class="top-bar__logo"
+            >
+          </a>
+          <div class="top-bar__icons">
+            <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
```

Comment:
Bad value tel:+1 234 5555-55-55 for attribute href on element [a](https://html.spec.whatwg.org/multipage/#the-a-element): Illegal character in scheme data: space is not allowed.